runtime/src/lib.rs

```diff
@@ -284,6 +314,9 @@ construct_runtime!(
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage},
 		Sudo: pallet_sudo::{Pallet, Call, Config<T>, Storage, Event<T>},
 		Parentchain: pallet_parentchain::{Pallet, Call, Storage},
+		RandomnessCollectiveFlip: pallet_randomness_collective_flip,
```

Comment:
Why are you not using the Pallet & Storage of this pallet? (https://github.com/ajuna-network/ajuna-node/blob/master/runtime/src/lib.rs#L288)